### PR TITLE
Add sofa branch v22.12.

### PR DESCRIPTION
**Pull request by olivier-roussel**
I need it for v22.12 binaries, necessary for the ongoing Conda package.

### CODE REVIEW

**Comment by damienmarchal** [comment]
Ok to me.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macos-10.15, windows-2019]
-        sofa_branch: [master]
+        sofa_branch: [master, v22.12]
```
Comment (alxbilger) [inline .github/workflows/ci.yml:16]: Does it still make sense to rely on the `master` branch?

Comment (damienmarchal) [reply .github/workflows/ci.yml:16]: You mean the workflow should be v22.12 and once released, master back ?

Comment (alxbilger) [reply .github/workflows/ci.yml:16]: My bad, I thought the PR was to be merged in the v22.12 branch. I did not see that the v22.12 branch was edited directly

**Review by alxbilger — APPROVED** [review]

**Comment by alxbilger** [comment]
Now that it is merged in master, should you remove the `master` branch on the CI in the v22.12 branch?

**Pull request merged**